cspell.json

```diff
@@ -31,40 +25,35 @@
         "Bucketwheat",
         "Capitan",
         "Carron",
-        "Checkboxes",
```

Comment:
"Checkboxes" is a valid word. Is included it unnecessary now?

Reply:
Yeah, I went through the words and confirmed the ones that now exist in the regular dictionaries now